components/List/List.js

```diff
@@ -1,16 +1,30 @@
-import { StyledList, StyledItem } from "@/styles";
+import { StyledList, StyledItem, StyledDeleteButton } from "@/styles";
+
+export default function List({ transactions, setTransactions }) {
+  function deleteTransaction(id) {
+    const updatedTransactions = transactions.filter(
+      (transaction) => transaction.id !== id
+    );
+    setTransactions(updatedTransactions);
+  }
 
-export default function List({ transactions }) {
   return (
     <>
       <StyledList>
-        {transactions.map((transaction) => (
-          <StyledItem key={transaction.id}>
-            {transaction.date} - {transaction.amount} EUR -{" "}
-            {transaction.category} - {transaction.type} -{" "}
-            {transaction.description}
-          </StyledItem>
-        ))}
+        <>
+          {transactions.map((transaction) => (
+            <StyledItem key={transaction.id}>
+              {transaction.date} - {transaction.amount} EUR -{" "}
+              {transaction.category} - {transaction.type} -{" "}
+              {transaction.description}{" "}
+              <StyledDeleteButton
+                onClick={() => deleteTransaction(transaction.id)}
+              >
+                ❌
+              </StyledDeleteButton>
+            </StyledItem>
+          ))}
+        </>{" "}
```

Comment:
```suggestion
        </>
```
I'm assuming you don't need this. :D